docs/source/installation.rst

```diff
@@ -16,29 +15,27 @@ For more information on installing collections, see `using collections`_.
 
 Ansible Galaxy
 --------------
-
-You can use the `ansible-galaxy`_ command with the option ``install`` to
-install a collection on your system (control node) hosted in Galaxy.
-
 Galaxy enables you to quickly configure your automation project with content
-from the Ansible community. Galaxy provides prepackaged units of work known as
-collections. If you have installed a prior version you may need to force
-overwriting an existing collection with the ``--force`` option.
+from the Ansible community.
 
-Here are examples for installing the **IBM z/OS core collection**:
+Galaxy provides prepackaged units of work known as collections. You can use the
+`ansible-galaxy`_ command with the option ``install`` to install a collection on
+your system (control node) hosted in Galaxy. If you have installed a prior
+version, you must overwrite an existing collection with the ``--force`` option.
+
+Here are a few examples of installing the **IBM z/OS core collection**:
 
 .. code-block:: sh
 
    $ ansible-galaxy collection install ibm.ibm_zos_core\
    $ ansible-galaxy collection install -f ibm.ibm_zos_core
    $ ansible-galaxy collection install --force ibm.ibm_zos_core
 
-
-By default, ansible-galaxy installs the latest collection that is available but
-you can add a version identifier to install a specific version. When installing
-a collection from Galaxy, it would be good to review the versions available as
-there can be point releases, mods and beta versions available and you may be
-interested in features in a particular version.
+By default, the `ansible-galaxy`_ command installs the latest available
+collection, but you can add a version identifier to install a specific version.
+Before installing a collection from Galaxy, review all the available versions.
+There could be new patch releases and beta versions available and you might
```

Comment:
@RadhaV2020  i think we can edit "There could be new patch releases and beta versions available and you might
there can be point releases, mods and beta versions available and you may be
be interested in the features in a particular version" to

"Periodically new releases become available containing enhancements and features you might be interested in."

Think we can remove mods, patch, etc just overly detailed for no reason.

Reply:
sounds good. Will update.